src/controllers/authController.ts

```diff
@@ -0,0 +1,50 @@
+import { Request, Response } from "express";
+import bcrypt from "bcrypt";
+import jwt from "jsonwebtoken";
+import User from "../models/User"; // Adjust the path to your User model
+import { validationResult } from "express-validator";
+
+export const registerUser = async (req: Request, res: Response): Promise<void>  => {
+    
+    console.log("jjjjjj");
+  try {
+    const errors = validationResult(req);
```

Comment:
I think we have this as a middleware in the validation file, cant you simply pass it to the rout in the routs.ts?